backend/src/agents/__init__.py

```
@@ -200,6 +251,57 @@ def init_backend(runtime: ToolRuntime, *, routes):
+
+        now_iso = datetime.now(tz_mod.utc).isoformat()
+        lines.append(f"CURRENT_UTC: {now_iso}")
+    if timezone_val:
+        lines.append(f"TIMEZONE: {timezone_val}")
+    if lang:
+        lines.append(f"LANGUAGE: {lang}")
+    return lines
+
+
+def _build_agents_md_content(
+    system_prompt: str, config: dict, instructions: str = None
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)